src/handle-data.js

```diff
@@ -50,6 +50,11 @@ export function getColumnInfo(layout, colIndex, columnOrder) {
       sortDirection: directionMap[info.qSortIndicator],
       dataColIdx: colIndex,
       totalInfo: getTotalInfo(isDim, layout, colIndex, numDims, columnOrder),
+      representation: {
```

Comment:
Is this needed for the example or part of some other feature?

Reply:
it's needed for charts inside the table